src/mumble/MainWindow.cpp

```diff
@@ -1379,6 +1370,35 @@ void MainWindow::setOnTop(bool top) {
 	}
 }
 
+void MainWindow::applyState(const bool minimalView) {
+	if (minimalView) {
+		if (!Global::get().s.qbaMinimalViewGeometry.isNull()) {
+			restoreGeometry(Global::get().s.qbaMinimalViewGeometry);
+		}
+		if (!Global::get().s.qbaMinimalViewState.isNull()) {
+			restoreState(Global::get().s.qbaMinimalViewState, STATE_VERSION);
+		}
+	} else {
+		if (!Global::get().s.qbaMainWindowGeometry.isNull()) {
+			restoreGeometry(Global::get().s.qbaMainWindowGeometry);
+		}
+		if (!Global::get().s.qbaMainWindowState.isNull()) {
+			restoreState(Global::get().s.qbaMainWindowState, STATE_VERSION);
+		}
+	}
+}
+
+void MainWindow::storeState(const bool minimalView) {
+	if (minimalView) {
+		Global::get().s.qbaMinimalViewGeometry = saveGeometry();
+		Global::get().s.qbaMinimalViewState    = saveState(STATE_VERSION);
+	} else {
+		Global::get().s.qbaMainWindowGeometry = saveGeometry();
+		Global::get().s.qbaMainWindowState    = saveState(STATE_VERSION);
+		Global::get().s.qbaHeaderState        = qtvUsers->header()->saveState();
```

Comment:
Here we are saving the state of the userview header, but it doesn't look like we ever apply it.

Reply:
Then I'd vote to throw it out

Reply:
Separate commit seems sufficient to me